src/psadt-silent.template/templates/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$ErrorActionPreference = 'Stop' # stop on all errors
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$destination = (Join-Path $toolsDir -ChildPath $env:ChocolateyPackageName)
+
+
+$adtExe = Join-Path $destination -ChildPath "Deploy-Application.exe"

```

Comment:
Does this `deploy-application.exe` needed to be excluded from the package reducer?